### ISSUE TITLE
Retire hack to fix RTC issues in Pokémon GSC and enhancement suggestion

### ISSUE THREAD

**Issue by SatoMew** [visualboyadvance-m/visualboyadvance-m#615]
I'm referring to this thread: https://board.vba-m.com/showthread.php?tid=707

In its place, create a new thread advising players to use PKHeX to correctly set the byte in the save file which is normally only touched if the clock data is not sane since it supports all variants of the save data (Japanese, Western, and Korean).

Also, is the information [here](http://bgb.bircd.org/rtcsave.html) regarding VBA-M still accurate? If so, could VBA-M revise its implementation for storing the RTC data in the save file? If you could optionally allow to store the RTC data in a separate file like other emulators, that would be neat.

**Comment by Squall-Leonhart** [comment]
No we cannot revise the save implementation, since its  the save implementation used by every other sane emulator and flash cart.

**Comment by rkitover** [comment]
I would agree that we have to preserve file format compatibility to the greatest extent possible, but tweaks may still be possible, like when I made 32bit and 64bit saves compatible a few months ago.

Optionally making another file is one of the things we could do, because that does not affect compatibility.
